lib/obsolete.gd

```diff
@@ -171,12 +171,18 @@ BindGlobal( "DeclarePackageAutoDocumentation", Ignore );
 # 03/2018: not used in any redistributed package
 
 DeclareObsoleteSynonym( "ReadPkg", "ReadPackage" );
-# 03/2018: still used in automgrp, ctbllib, fplsa, fwtree, Hap (HapCocyclic),
+# 03/2018: still used in automgrp, fplsa, fwtree, Hap (HapCocyclic),
 # itc, modisom, pargap, quagroup
+# (Note that 'grep' finds 'ReadPkg' also in 'ctbllib/init.g',
+# but that line of code is reached only if the file is read in GAP 3.
+# This occurrence of 'ReadPkg' will remain forever.)
```

Comment:
Coincidentally, I've been preparing a PR which updates obsolete.gd, and which also addresses this bit about ctbllib (but also much more).

Could I convinced you to drop this here? Then we don't have two conflicting PRs.